app/Http/Requests/StoreAppendBundleRequest.php

```diff
@@ -3,6 +3,7 @@
 namespace App\Http\Requests;
 
 use Illuminate\Foundation\Http\FormRequest;
+use App\Voucher;
```

Comment:
:smile: You know me - I like these to be ABC - but not stopping you. Hardly relevant with 2 imports!